torchvision/csrc/cpu/image/read_write_file_cpu.cpp

```diff
@@ -1,10 +1,17 @@
 #include "read_write_file_cpu.h"
 
+// According to
+// https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?view=vs-2019,
+// we should use _stat64 for 64-bit file size on Windows.
+#ifdef _WIN32
+#define VISION_STAT _stat64
+#else
+#define VISION_STAT stat
```

Comment:
nit: I wonder if best practices for macros is to  `#undef` those after they are used

Reply:
Strictly speaking, yes. But it is okay since it is in a source file.